src/VPolytope.jl

```diff
@@ -385,16 +386,25 @@ Return the polytope obtained by removing the redundant vertices of the given pol
                computations backend, see
                [Polyhedra's documentation](https://juliapolyhedra.github.io/Polyhedra.jl/latest/installation.html#Getting-Libraries-1)
                for further information
+- `solver`  -- (optional, default: `default_lp_solver(N)`) the linear programming
+               solver used in the backend
 
 ### Output
 
 A new polytope such that its vertices are the convex hull of the given polytope.
 """
 function remove_redundant_vertices(P::VPolytope{N};
-                                   backend=default_polyhedra_backend(P, N))::VPolytope{N} where {N<:Real}
+                                   backend=default_polyhedra_backend(P, N),
+                                   solver=default_lp_solver(N))::VPolytope{N} where {N<:Real}
     require(:Polyhedra; fun_name="remove_redundant_vertices")
+
     Q = polyhedron(P; backend=backend)
-    removevredundancy!(Q)
+    if Polyhedra.supportssolver(typeof(Q))
+        vQ = Polyhedra.vrep(Q)
+        Polyhedra.setvrep!(Q, Polyhedra.removevredundancy(vQ, solver))
+    else
+        Polyhedra.removevredundancy!(Q)
```

Comment:
I do not understand what is going on here. Can you add some documentation?

Reply:
I have added the description in the docstring.